tests/conftest.py

```diff
@@ -38,16 +38,16 @@ def _drop_all():
     yield session
     session.close()
 
-    # _drop_all()
-
 
 @pytest.fixture(scope="function", name="app")
 def app_fixture(engine):
-    return applications.FastAPI_SQLAlchemy(
-        engine,
+    app = applications.FastAPI_SQLAlchemy(
```

Comment:
See other comments - the FastAPI_SQLAlchemy object should become a helper instead of a requirement.

Reply:
Done